tests/trustyai/conftest.py

```diff
@@ -86,3 +95,123 @@ def minio_data_connection(
         },
     ) as minio_secret:
         yield minio_secret
+
+
+@pytest.fixture(scope="class")
+def db_credentials(admin_client, model_namespace: Namespace) -> Secret:
+    with Secret(
+        client=admin_client,
+        name="db-credentials",
+        namespace=model_namespace.name,
+        string_data={
+            "databaseKind": MARIADB,
+            "databaseName": "trustyai_database",
+            "databaseUsername": QUARKUS,
+            "databasePassword": QUARKUS,
+            "databaseService": MARIADB,
+            "databasePort": "3306",
+            "databaseGeneration": "update",
+        },
+    ) as db_credentials:
+        yield db_credentials
+
+
+@pytest.fixture(scope="session")
+def mariadb_operator(admin_client: DynamicClient) -> None:
+    name = "mariadb-operator"
+    namespace = "openshift-operators"
+    install_operator(
+        admin_client=admin_client,
+        target_namespaces=[namespace],
+        name=name,
+        channel="alpha",
+        source="community-operators",
+        operator_namespace=namespace,
+        timeout=600,
```

Comment:
please add a constant (10 * 60 is used as well)

Reply:
Done